Command/DocumentCommand.php

```diff
@@ -0,0 +1,208 @@
+    {
+        $this->contentTypeService = $contentTypeService;
+        $this->documentService = $documentService;
+        $this->dataService = $dataService;
+        $this->dataService = $dataService;
+        $this->logger = $logger;
+        $this->client = $client;
+        $this->ready = false;
+        parent::__construct();
+    }
+    
+    protected function configure()
+    {
+        $this
+            ->setDescription('Import json files from a zip file as content type\'s documents')
+            ->addArgument(
+                self::ARGUMENT_CONTENTTYPE,
+                InputArgument::REQUIRED,
+                'Content type name to import into'
+            )
+            ->addArgument(
+                'archive',
+                InputArgument::REQUIRED,
+                'The archive (zip file or directory) containing the json files'
+            )
+            ->addOption(
+                'bulkSize',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Size of the elasticsearch bulk request',
+                500
+            )
+            ->addOption(
+                'raw',
+                null,
+                InputOption::VALUE_NONE,
+                'The content will be imported as is. Without any field validation, data stripping or field protection'
+            )
+            ->addOption(
+                'dont-sign-data',
+                null,
+                InputOption::VALUE_NONE,
+                'The content will not be signed during the import process'
+            )
+            ->addOption(
+                'force',
+                null,
+                InputOption::VALUE_NONE,
+                'Also treat document in draft mode'
+            )
+            ->addOption(
+                'dont-finalize',
+                null,
+                InputOption::VALUE_NONE,
+                'Don\'t finalize document'
+            )
+            ->addOption(
+                'businessKey',
+                null,
+                InputOption::VALUE_NONE,
+                'Try to identify documents by their business keys'
+            );
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->io = new SymfonyStyle($input, $output);
+    }
+
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        $arguments = array_values($input->getArguments());
+        array_shift($arguments);
+        list($contentTypeName, $archiveFilename) = $arguments;
+
+        $this->io->title('Make documents');
+        $this->io->section('Checking input');
+
+
+        $contentType = $this->contentTypeService->getByName($contentTypeName);
+        if (!$contentType instanceof ContentType) {
+            $this->io->error(sprintf('Content type %s not found', $contentTypeName));
+            return;
```

Comment:
```suggestion
            return -1;
```

Reply:
Being explicit in the interact that execution is stopped completely at this point.

Reply:
It would be better to use the question interface to get a correct value (nice to have)